config/config.default.js

```diff
@@ -226,7 +226,7 @@ module.exports = appInfo => {
     agentLogName: 'egg-agent.log',
     errorLogName: 'common-error.log',
     coreLogger: {},
-    allowDebugAtProd: true,
```

Comment:
shouldn't change the default value here, it's a breaking change

Reply:
Is there any other better way?

Reply:
it's more seems like a bug fix, `allowDebugAtProd ` should be `false` by default,  since we don't want to log at prod